Furball.Engine/FurballGame.cs

```diff
@@ -183,43 +186,59 @@ public class FurballGame : Game {
 
             DebugOverlayDrawableManager.Add(_ConsoleAutoComplete);
 
-            WhitePixel = new Texture2D(this.GraphicsDevice, 1, 1);
-            Color[] white = { Color.White };
-            WhitePixel.SetData(white);
+            WhitePixel = new Texture();
 
             LocalizationManager.ReadTranslations();
 
             TooltipDrawable = new();
             DrawableManager.Add(TooltipDrawable);
 
+            ScreenManager.ChangeScreen(this._startScreen);
+
+            DrawableBatch = new DrawableBatch(RendererType.Batched);
+
+            this.ChangeScreenSize(DEFAULT_WINDOW_WIDTH, DEFAULT_WINDOW_HEIGHT);
+
+            DebugCounter = new DebugCounter {
+                Clickable   = false,
+                CoverClicks = false
+            };
+            DebugOverlayDrawableManager.Add(DebugCounter);
+
+            DevConsole.Initialize();
+
+            ScreenManager.SetTransition(new FadeTransition());
+
             base.Initialize();
 
             EtoHelper.Initialize();
         }
-
-        protected override void OnExiting(object sender, EventArgs args) {
+        protected override void OnClosing() {
             DevConsole.Run(":nt_on_exiting", false, true);
             DevConsole.WriteLog();
 
+            GameTimeScheduler.Dispose(Time);
             EtoHelper.Dispose();
 
-            base.OnExiting(sender, args);
+            base.OnClosing();
         }
 
         public void SetTargetFps(int fps, double unfocusedScale = -1) {
-            if (fps != -1) {
-                this.TargetElapsedTime = TimeSpan.FromMilliseconds(1000 / (double)fps);
-                this.IsFixedTimeStep   = true;
-            } else {
-                this.TargetElapsedTime = TimeSpan.FromTicks(1);
-                this.IsFixedTimeStep   = false;
-            }
+            WindowManager.SetTargetFramerate(fps);
+            //if (fps != -1) {
+            //    this.TargetElapsedTime = TimeSpan.FromMilliseconds(1000 / (double)fps);
+            //    this.IsFixedTimeStep   = true;
+            //} else {
+            //    this.TargetElapsedTime = TimeSpan.FromTicks(1);
+            //    this.IsFixedTimeStep   = false;
+            //}
```

Comment:
Are we not able to set uncapped anymore? or is that done in `WindowManager.SetTargetFramerate`

Reply:
vixie doesnt do this anymore, maybe i removed this for testing? or during the time where u couldnt set framerate, ill readd this

Reply:
readded now